### ISSUE TITLE
Remove requirement for `*.test.js` naming when file is explicitly called

### ISSUE THREAD

**Issue by erquhart** [galenframework/galen#166]
The `*.test.js` naming scheme isn't unique, and I'd like to replace it with `*.galen.js` for our project. The only real change needed here is to not require the `*.test.js` naming if a specific file (not a directory) is initially called in the cli:
```
// should work:
galen test home.galen.js
```

Basically a manual override - if I'm telling Galen to test a specific file, it should be assumed that it's the proper file type.

**Comment by ishubin** [comment]
I am going to implement it as a config setting but will leave the default behaviour of "*.test.js". If you want to have a different pattern - you will have to configure it in config file or pass it as a system property

**Comment by ishubin** [comment]
Fixed in release-1.6 branch.
From now on, if you just specify the test file directly in command line, it has to only end with ```.js``` suffix: 
```
galen test myTests/loginPageTest.js
```

And also if you want to recursively find all tests inside a folder with a different suffix, you can specify the config property ```galen.test.js.file.suffix = .galen.js```
